stdlib/public/Concurrency/Task.swift

```diff
@@ -76,6 +76,45 @@ import Swift
 /// like the reason for cancellation.
 /// This reflects the fact that a task can be canceled for many reasons,
 /// and additional reasons can accrue during the cancellation process.
+///
+/// ### Task closure lifetime
+/// Tasks initialized with a closure, representing the code that will be executed by this task.
```

Comment:
How about:

```suggestion
/// A task is initialized with a closure. containing the code that the task executes.
```

More direct wording (closures don't "represent" code) and avoiding future tense per Apple Style Guide.